vg/vgx11/canvas.go

```diff
@@ -4,7 +4,7 @@
 
 // +build !windows
 
-// The vgx11 package uses xgbutil (github.com/BurntSushi/xgbutil)
+// Package vgx11 uses xgbutil (github.com/BurntSushi/xgbutil)
```

Comment:
Sorry, I'd really like this to be "Package vgx11 implements X11 vg support." or words to that effect. The basis for the implementation is secondary and should not be in the first sentence.